### ISSUE TITLE
Crash with only Enviromine

### ISSUE THREAD

**Issue by sweetsosweet** [EnviroMine/EnviroMine-1.7#178]
Hi, this is my first time trying Enviromine, so I've downloaded the latest release version on curse, unfortunately it crashes when I create a new world, even with only Enviromine. Here is my crashlog:

http://pastebin.com/KMf9rFGL

I have since downgraded to the previous version instead, where I noticed a folder called CustomProperties was created. This folder wasn't created in the latest version, just thought I'd point that out.

**Comment by PitchBright** [comment]
Can you post the whole crash log please?

**Comment by sweetsosweet** [comment]
That is the whole crash log. Like I said this is only with enviromine installed. I think the new version is just not creating the initial files. When I installed the previous one the files were created, then I was able to play with the latest one with no issue. I think this would only affect new users that don't have previous data.

**Comment by PitchBright** [comment]
That's the entire contents of the file? There should be something somewhere which shows us the version of stuff being used. Usually at the bottom. (You say "latest" version but we have no way of telling if you are in fact using the latest version.)

I took a look in the configs for ya, because I remembered there was this stuff:

Config {
    # [DISABLED][WIP] Temporarily overrides client configurations with the server's (NETWORK INTESIVE!)
    B:"Client Config Override (SMP)"=false

    # Will attempt to find and generate blank configs for any custom items/blocks/etc loaded before EnviroMine. Pack developers are highly encouraged to enable this! (Resets back to false after use)
    B:"Generate Blank Configs"=false

    # Generates EnviroMines initial default files
    B:"Generate Defaults"=false
}

but your problem is those configs aren't getting created. :-(

**Comment by sweetsosweet** [comment]
Are you unable to scroll pastebin? It says right there in the file:

UCHIJAA enviromine{1.3.123} [EnviroMine] (EnviroMine-1.3.123.jar)

**Comment by PitchBright** [comment]
Ah ya, there it is thanks. I was on my ipad at the time, so it was either that, or the fact that I had just woken up that caused me to miss it.  xD

**Comment by sweetsosweet** [comment]
:D I figured it might be something like that. But yeah, I messed around with those exact configs too, and no dice. It just kept crashing. But like I mentioned, I have downloaded version 1.3.122 and allowed that to generate the files before updating to 1.3.123 again.

**Comment by PitchBright** [comment]
Sounds like you've pinpointed it as a bug, so I hope they get on here and label this legit. My guess is something got deactivated by accident in the latest version, and might be an easy fix for them.

**Comment by sweetsosweet** [comment]
Yup, that's what I'm thinking. Thanks for taking the time to respond. :)
